nfpm/zrok-share.bash

```diff
@@ -54,9 +54,12 @@ fi
   exit 1
 }
 
-# default mode is reserved (public), override mode is temp-public, i.e., "share public" without a reserved subdomain
+# default mode is 'temp-public' (unreserved), override modes are temp-private, reserver-public, reserved-private.
+: "${ZROK_FRONTEND_MODE:-temp-public}"
```

Comment:
I'd like to preserve the default mode as `reserved-public` because the predictable, long-lived share token is generally more useful, and for historical reasons, the script was conceived to support the Linux package where a reserved token is almost always used.

Reply:
This sounds good to me. Certainly my intention was to preserve existing behaviour.